src/modules/ProductLicense.rb

```diff
@@ -1601,6 +1613,15 @@ def update_license_location(lang, licenses)
       end
     end
 
+    # update license location displayed in the dialog
+    # @param [Fixnum] src_id integer repository to get the license from.
+    def update_license_archive_location(src_id)
+      src_url = Pkg::SourceGeneralData(src_id)["url"]
+      if location_is_url?(src_url) && UI.WidgetExists(:printing_hint)
+        lic_url = src_url + @license_file_print
```

Comment:
I'd use `File.join` here to properly handle '/' separator when merging the parts.